tests/e2e/README.md

```diff
@@ -1,37 +1,74 @@
 # WooCommerce Payments End-to-end tests
 
-E2E tests can be run locally or in GitHub Action.
+E2E tests can be run locally or in GitHub Actions. Github Actions are already configured and doesn't require any changes to run the tests.
 
-## Setup
+## Setting up & running E2E tests
 
-Setup script requires the following env variables to be configured:
+For running E2E tests locally, create a new file named `local.env` under `tests/e2e/config` folder with the following env variables (replace values as required).
+
+<details>
+<summary>Required env variables</summary>
+<p>
 
 ```
 WCP_SERVER_REPO='https://github.com/server-repo.git or git@github.com:org/server-repo.git'
 WCP_DEV_TOOLS_REPO='https://github.com/dev-tools-repo.git or git@github.com:org/dev-tools-repo.git'
 
+// Optional to see verbose output
+DEBUG=true
+```
+
+</p>
+</details>
+
+---
+
+<details>
+<summary>Choose WCPay Server instance</summary>
+<p>
+
+It is possible to use the live server or a local docker instance of WCPay server locally. On Github Actions, live server is used for tests. Add the following env variables to your `local.env` based on your preference (replace values as required).
+
+**Using Local Server on Docker**
+```
+E2E_USE_LOCAL_SERVER=true
```

Comment:
Could we use this as true by default so that folks don't have to change their local config after pulling in these changes?

Reply:
Sure @deepakpathania I'll update the docs